ops/migrations_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# This script is intended to be run from within the Docker "unit_test" container
+# The docker-compose file brings forward the env vars: DB
+#
+set -e
+set -o pipefail
+
+timestamp=`date +%Y-%m-%d_%H-%M-%S`
+
+echo "Setting up test DB (bcda_test)..."
```

Comment:
Should the test db be named `ssas_test` in this env?

Reply:
Probably.  But the main database is still `bcda`, so I'm not sure whether `bcda_test` is more congruent.

Reply:
Yeah, makes sense to leave it as is.